.github/workflows/autobuild.yml

```diff
@@ -54,17 +54,13 @@ jobs:
       - name: Set up git repository
         run: git config --global --add safe.directory "$GITHUB_WORKSPACE"
 
-      - name: Include custom build template instead of package default
-        run: |
-          cp -v anda/terra/mock-configs/terra.tpl /etc/mock/templates/terra.tpl
-
       - name: Build with Andaman
-        run: anda build ${{ matrix.pkg.pkg }} --package rpm -c anda/terra/mock-configs/terra-${{ matrix.version }}-${{ matrix.pkg.arch }}.cfg
+        run: anda build ${{ matrix.pkg.pkg }} --package rpm -c terra-${{ matrix.version }}-${{ matrix.pkg.arch }}
```

Comment:
If for some reason mock configs are broken, this will make rebuilding mock configs a very difficult task
Maybe specifically clone the mock config sources and use it here?